addons/storyshots/storyshots-core/package.json

```diff
@@ -61,7 +61,7 @@
     "jest-specific-snapshot": "^4.0.0",
     "preact-render-to-string": "^5.1.19",
     "pretty-format": "^26.6.2",
-    "react-test-renderer": "^16.8.0 || ^17.0.0",
```

Comment:
I think we need a devDependency on the version we're currently using in the monorepo; otherwise yarn might install a too new of a version if we regenerate the lockfile.

Could you add this?

Reply:
Looks like the unit test are failing because of the update, could you have a look?
https://app.circleci.com/pipelines/github/storybookjs/storybook/25536/workflows/5f9b2073-68d0-44cc-b528-962a226d74a3/jobs/378122 
🙏

Reply:
@ndelangen Thanks for your feedback!

> I think we need a devDependency on the version we're currently using in the monorepo; otherwise yarn might install a too new of a version if we regenerate the lockfile.
> 
> Could you add this?

Did you mean that I should move `react-test-renderer` in `dependencies` to `devDependencies`?

Reply:
Yes, the unit test is failing because of these `'isBatchingLegacy' of undefined` errors. 

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/19849611/166677052-85153b47-116e-403f-9257-134843f02f1f.png">

Maybe I should first check what change has been made to `react-test-renderer` changed since version 17?
https://github.com/facebook/react/commits/main/packages/react-test-renderer

Reply:
Looks like react-test-renderer 18 is not meant to be used with older react.
We probably cannot be able to upgrade react-test-renderer before upgrading react and react-dom in this repo first 🤔